recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
+{% set sha256 = "a14c20c61c086414895142d22d4ba9b4a5267a743dddfff860d947a5126fdb79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-fixed.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-fixed.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.7*
```

Comment:
Yes, I believe the latest version does support python3 (and I will update this) but 0.7.9 doesn't.

I just tried implementing your suggestion, but when I do, building with
`conda build --python 2.7 plumpy`
yields syntax errors like
```
running install_scripts
writing list of installed files to 'record.txt'
INFO:conda_build.build:Packaging plumpy-0.7.9-py27_0
compiling .pyc files...
  File "lib/python3.6/site-packages/plum/engine/ticking.py", line 81
    raise type(self._exception), self._exception, self._traceback
````
I'm not quite sure why it is using my default python version (3.6) in this case - perhaps it is building *for* python27 *with* python36?